pkg/pgmodel/querier/series_set.go

```diff
@@ -81,10 +97,21 @@ func (p *pgxSeriesSet) At() storage.Series {
 	// this should pretty much always be non-empty due to __name__, but it
 	// costs little to check here
 	if len(row.labelIds) != 0 {
-		lls, err := p.querier.LabelsForIds(row.labelIds)
-		if err != nil {
-			log.Error("err", err)
-			return nil
+		var lls labels.Labels
+		lls = make([]labels.Label, 0, len(row.labelIds))
```

Comment:
```suggestion
		lls := make([]labels.Label, 0, len(row.labelIds))
```

Reply:
can't do this since want labels.Labels not []labels.Label